parts/windows/kuberneteswindowsfunctions.ps1

```diff
@@ -58,7 +58,7 @@ function DownloadFileOverHttp {
         }
 
         $ProgressPreference = $oldProgressPreference
-        Write-Log "Downloaded file to $DestinationPath"
+        Write-Log "Downloaded file $fileName to $DestinationPath"

```

Comment:
What about `Downloaded file $Url to $DestinationPath`, as the URL contains the file name, and both variables are function inputs

Reply:
Use $fileName to keep the same format as above `Using cached version of $fileName`. Agree that $Url is better.